common/autoware_lanelet2_ros_interface/CMakeLists.txt

```diff
@@ -32,98 +30,174 @@ find_path(PUGIXML_INCLUDE_DIRS
   PATH_SUFFIXES pugixml
 )
 
-find_package(autoware_build_flags REQUIRED)
-
-find_package(PROJ4 REQUIRED)
-
-find_package(catkin REQUIRED COMPONENTS
-  amathutils_lib
-  roscpp
-  lanelet2_core
-  lanelet2_io
-  lanelet2_maps
-  lanelet2_projection
-  lanelet2_routing
-  lanelet2_traffic_rules
-  lanelet2_validation
-  lanelet2_extension
-  autoware_lanelet2_msgs
-  autoware_msgs
-  geometry_msgs
-  visualization_msgs
-  roslint
-)
 
-set(ROSLINT_CPP_OPTS "--filter=-build/c++14")
-roslint_cpp()
+if(${ROS_VERSION} EQUAL 1)
+  message("Building ros1")
```

Comment:
If you want to keep this message then it should be set to the debug log level

Reply:
I can remove these, they dont seem helpful merged in